callstack/react-native-builder-bob
# feat(bob): support custom target definitions

### Summary

This adds the `custom` target to bob. Users are able to pass arbitrary scripts via this target and bob will call those scripts.

### Test plan

1. Configure bob in a new project
2. Define the `custom` target
3. Define a script that generates some files
4. Call bob build and make sure the script was called with the right package manager
